: NOISSUE: Update the LB service document by using a free host port (#1588)
openshift/microshift: docs/howto_load_balancer.md

  name: nginx
spec:
  ports:
  - port: 81
    targetPort: 8080
  selector:
    app: nginx
  type: LoadBalancer
EOF
```

> > **_NOTE:_** `port` shall be a free host port that is not occupied by other
> > `LoadBalancer` services, host processes or Microshift components.

Reviewer: ```suggestion
> The `port` setting should be a host port that is not occupied by other `LoadBalancer` services,
> host processes or MicroShift components.
```
